docs/source/conf.py

```diff
@@ -0,0 +1,180 @@
+# -*- coding: utf-8 -*-
+#
+# Configuration file for the Sphinx documentation builder.
+#
+# This file does only contain a selection of the most common options. For a
+# full list see the documentation:
+# http://www.sphinx-doc.org/en/master/config
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import os
+import sys
+sys.path.insert(0, os.path.abspath('../../'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = u'circulation-admin'
+copyright = u'2019, ed'
+author = u'ed'
+
+# The short X.Y version
+version = u''
+# The full version, including alpha/beta/rc tags
+release = u''
```

Comment:
Does this have to be update for every release in order to have docs for different releases? That would be annoying.

Reply:
It would be. There is a `stable` and `latest` convention on readthedocs. I imagine we would use `latest` for master` but `stable` for when we have a new tag release.